src/main/java/it/gov/pagopa/pu/registry/mapper/installment/DebtPositionEventDTO2InstallmentRegistryMapper.java

```diff
@@ -2,19 +2,24 @@
 
 import it.gov.pagopa.pu.registry.event.payments.dto.DebtPositionEventDTO;
 import it.gov.pagopa.pu.registry.model.InstallmentRegistry;
+import it.gov.pagopa.pu.registry.utils.ExtractionUtils;
 import org.springframework.stereotype.Service;
 
 import java.util.List;
+import java.util.Set;
 
 @Service
 public class DebtPositionEventDTO2InstallmentRegistryMapper {
 
   public List<InstallmentRegistry> map(DebtPositionEventDTO dto) {
     if (dto.getPayload().getPaymentOptions() == null) return List.of();
 
+    Set<String> iudIds = ExtractionUtils.extractIudIdsFromDescription(dto.getEventDescription());
```

Comment:
IUD = IdentificativoUnivoco(Dominio?)
iudIds is redundant
```suggestion
    Set<String> iuds = ExtractionUtils.extractIudIdsFromDescription(dto.getEventDescription());
```

Reply:
ps: let's rename also the utility function :D
extractIudsFromDescription